doc/api/events.md

```diff
@@ -1277,11 +1277,13 @@ added: REPLACEME
 
 Removes the `listener` from the list of handlers for event `type`.
 
-### Class: `NodeEventTarget extends EventTarget`
+### Class: `NodeEventTarget`
```

Comment:
The addition of the `* Extends:` line below is good, but to be honest I'd much prefer if we started adding the `extends ...` in the header line also as a best practice to make the extension more visible.